CompanyEmployees.Presentation/Controllers/EmployeesController.cs

```diff
@@ -22,11 +23,22 @@ public IActionResult GetEmployeesForCompany(Guid companyId)
             return Ok(employees);
         }
 
-        [HttpGet("{id:guid}")]
+        [HttpGet("{id:guid}", Name = "GetEmployeeForCompany")]
         public IActionResult GetEmployeeForCompany(Guid companyId, Guid id)
         {
             var employee = _service.EmployeeService.GetEmployee(companyId, id, trackChanges: false);
             return Ok(employee);
         }
+
+        [HttpPost]
+        public IActionResult CreateEmployeeForCompany(Guid companyId, [FromBody] EmployeeForCreationDto employee)
+        {
+            if (employee is null)
+                return BadRequest("EmployeeForCreationDto object is null");
```

Comment:
This DTO can simply be named "EmployeeCreateDto"

Reply:
I have renamed both Employee and Company Dtos to EmployeeCreateDto and CompanyCreateDto respectively.